src/components/result/PriceDialog/index.tsx

```diff
@@ -1,137 +1,144 @@
 import { useMedia, Divider } from '@offer-ui/react'
-import type { ReactElement, MouseEvent } from 'react'
+import type { ReactElement, MouseEvent, ChangeEvent } from 'react'
 import { useState } from 'react'
 import { Styled } from './styled'
-import type { PriceDialogProps } from './types'
-import { toLocaleCurrency } from '@utils'
+import type { PriceDialogProps, PriceRange } from './types'
+import { useModal } from '@hooks/useModal'
+import { localeCurrencyToNumber, toLocaleCurrency } from '@utils'
+
 const PriceDialog = ({
-  inputPrice,
-  applyPrice,
-  handlePriceInputChange,
-  handlePriceApplyClick
+  priceRange: defaultPriceRange,
+  onClickApply
 }: PriceDialogProps): ReactElement => {
-  const [isOpen, setIsOpen] = useState<boolean>(false)
+  const [priceRange, setPriceRange] = useState<PriceRange>(defaultPriceRange)
+  const priceDialog = useModal()
   const { desktop } = useMedia()
 
-  const isApplied = applyPrice?.maxPrice && applyPrice?.minPrice
+  const isApplied = defaultPriceRange?.max && defaultPriceRange?.min
```

Comment:
```suggestion
  const isApplied = defaultPriceRange?.max && defaultPriceRange?.min
```
withRange, isAppliedRange 는 어떠신가요? 변수명만 보고 유추하기에는 어려움이 있을 것 같습니다!

Reply:
[변경 완료](https://github.com/price-offer/offer-fe/pull/238/commits/47819e40005624da1b2f6c299f1a0ed1875a98a1#diff-7bfaf01862b5d7346d9ab6504ef1e368e31494ddd9a8d21d08e313883d503618R17) !